Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/SceneShortInfo.cs

```diff
@@ -16,6 +17,15 @@ public SceneShortInfo(Vector2Int baseParcel, string name)
             Name = name;
         }
 
+        public bool Equals(SceneShortInfo other) =>
```

Comment:
I am not sure why you added overloads but I advise to add `IEquatable` interface so if it is used as a dictionary (or hashset) key there will be no allocations

Reply:
you are right, not needed, I just removed it